src/DatePicker/DatePicker.js

```diff
@@ -350,6 +375,16 @@ class DatePicker extends Component {
             className
         );
 
+        const datepickerFooterClassName = classnames(
+            'fd-dialog__footer',
+            'fd-bar',
+            'fd-bar--footer',
+            {
+                'fd-bar--cosy': !compact,
+                'fd-bar--compact': compact
+            }
+        );
```

Comment:
should we add a prop so a consumer can add their own className to this?  `footerClassName`?